nano/rpc_test/rpc.cpp

```diff
@@ -1858,16 +1858,148 @@ TEST (rpc, pending)
 		ASSERT_EQ (block4->hash (), hash);
 		ASSERT_EQ (block3->hash (), hash1);
 	}
+}
+
+TEST (rpc, receivable_offset)
+{
+	nano::system system;
+	auto node = add_ipc_enabled_node (system);
+	nano::keypair key1;
+	system.wallet (0)->insert_adhoc (nano::dev::genesis_key.prv);
+	auto block1 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 200));
+	auto block2 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 100));
+	auto block3 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 400));
+	auto block4 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 300));
+	auto block5 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 300));
+	auto block6 (system.wallet (0)->send_action (nano::dev::genesis_key.pub, key1.pub, 300));
+	node->scheduler.flush ();
+	ASSERT_TIMELY (5s, node->ledger.account_pending (node->store.tx_begin_read (), key1.pub) == 1600);
+	ASSERT_TIMELY (5s, !node->active.active (*block6));
+	ASSERT_TIMELY (5s, node->block_confirmed (block6->hash ()));
+	ASSERT_TIMELY (5s, node->ledger.cache.cemented_count == 7 && node->confirmation_height_processor.current ().is_zero () && node->confirmation_height_processor.awaiting_processing_size () == 0);
+
+	auto const rpc_ctx = add_rpc (system, node);
+	boost::property_tree::ptree request;
+	request.put ("action", "receivable");
+	request.put ("account", key1.pub.to_account ());
+
+	// Default: Sorted by hash
```

Comment:
The documentation does not say that the results of the "receivable" rpc commands are sorted by hash.
The fact that the current implementation appears to return them sorted by hash (this is probably because they are sorted in the database) does not mean that this a feature and therefore we should not check that results are sorted by hash when sorting is false.